### DISCUSSION TITLE
declaring etcd servers in patroni.yml when dcs_exists: true

### DISCUSSION THREAD

**Discussion by legrandlegrand** [autobase-tech/autobase#609] [Q&A] [answered]
Hello,
During my test of postgresql_cluster on a server with etcd already installed from yum repo,
I get error:

```
TASK [patroni : Start patroni service on the Master server] ********************
changed: [99.99.99.99]

TASK [patroni : Wait for port 8008 to become open on the host] *****************
fatal: [99.99.99.99]: FAILED! => {"changed": false, "elapsed": 120, "msg": "Timeout when waiting for 99.99.99.99:8008"}
```

no  host is populated in patroni.yml file:
```
etcd3:
  hosts:

bootstrap:
```

But host is defined in inventory
```
# if dcs_exists: false and dcs_type: "etcd"
[etcd_cluster]  # recommendation: 3, or 5-7 nodes
1.2.3.4
```

I have this in main.yml:
```
dcs_exists: true
dcs_type: "etcd"  # or 'consul'
```

Any idea about how to fix that ?
Thanks in advance

Regards
PAscal

**Comment by vitabaks** [comment, marked as answer]
hello @legrandlegrand 

```yaml
# if dcs_type: "etcd" and dcs_exists: true
patroni_etcd_hosts: []  # list of servers of an existing etcd cluster
#  - { host: "10.128.64.140", port: "2379" }
#  - { host: "10.128.64.142", port: "2379" }
#  - { host: "10.128.64.143", port: "2379" }
```
https://github.com/vitabaks/postgresql_cluster/blob/master/vars/main.yml#L71

if you are using your ETCD cluster, which is not controlled by `postgresql_cluster` automation, then you should remove IP addresses from the inventory and specify them in the **patroni_etcd_hosts** variable.

**Comment by legrandlegrand** [comment]
Great ! That’s it !
Thank you for this quick answer
